src/js/media-interaction-feedback.js

```diff
@@ -0,0 +1,110 @@
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import { Window as window, Document as document } from './utils/server-safe-globals.js';
+import { MediaUIAttributes } from './constants.js';
+
+const template = document.createElement('template');
+const playIcon =
+  '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path class="icon" d="M8 5v14l11-7z"/><path d="M0 0h24v24H0z" fill="none"/></svg>';
+const pauseIcon =
+  '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path class="icon" d="M6 19h4V5H6v14zm8-14v14h4V5h-4z"/><path d="M0 0h24v24H0z" fill="none"/></svg>';
+
+template.innerHTML = `
+<style>
+:host {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1;
+  pointer-events: none;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+/*
+  Toggle between showing the play and pause icons
+  on the correct interactions
+*/
+:host([${MediaUIAttributes.MEDIA_PAUSED}]) slot[name=play] > *, 
+:host([${MediaUIAttributes.MEDIA_PAUSED}]) ::slotted([slot=play]) {
+  display: none;
+}
+
+:host(:not([${MediaUIAttributes.MEDIA_PAUSED}])) slot[name=pause] > *, 
+:host(:not([${MediaUIAttributes.MEDIA_PAUSED}])) ::slotted([slot=pause]) {
+  display: none;
+}
+
+/*
+  Visually hide the interaction when current time begins with 0.
+  This prevents the interaction from showing on initial load and
+  on initial play.
+*/
+:host([${MediaUIAttributes.MEDIA_CURRENT_TIME}^="0"]) slot[name=play] > *,
```

Comment:
This is a clever way to block on firstplay. It does make it so it doesn't work for the full first second of video. So if you play then immediately pause, then play again, it doesn't work. I think that's actually fine. And I'm assuming using `="0"` doesn't work because it's immediately not equal to zero.